sklearn/linear_model/tests/test_ridge.py

```diff
@@ -311,6 +311,96 @@ def test_ridge_individual_penalties():
     assert_raises(ValueError, ridge.fit, X, y)
 
 
+@pytest.mark.parametrize('gcv_mode', ['svd', 'eigen'])
+@pytest.mark.parametrize('X_constructor', [np.asarray, sp.csr_matrix])
+@pytest.mark.parametrize('X_shape', [(11, 8), (11, 20)])
+@pytest.mark.parametrize('fit_intercept', [True, False])
+@pytest.mark.parametrize('normalize', [True, False])
+@pytest.mark.parametrize('noise', [1., 30.])
+def test_ridge_gcv_vs_k_fold(gcv_mode, X_constructor, X_shape, fit_intercept,
```

Comment:
Because this is not k-fold CV, may I suggest instead:

```suggestion
def test_ridge_gcv_vs_ridge_loo_cv(gcv_mode, X_constructor, X_shape, fit_intercept,
```